src/DataCollector/SeoCollector.php

```diff
@@ -168,8 +168,12 @@ public function getTitle(): ?string
         return $this->data['title'];
     }
 
-    public function getTitleTruncate(): ?string
+    public function getTruncatedTitle(): ?string
     {
+        if (\is_null($this->data['title'])) {
```

Comment:
```suggestion
        if (null === $this->data['title']) {
```

(using `is_null` , i.e a function call instead of a simple check, is not much useful and uncommon) (same everywhere)

Reply:
😬